### ISSUE TITLE
Support the BFloat16 vector data type

### ISSUE THREAD

**Issue by roji** [milvus-io/milvus-sdk-csharp#76]
See https://milvus.io/docs/release_notes.md

**Comment by roji** [comment]
Makes sense to me! Changing the issue title to make clear that it's only for bfloat16, to gather feedback from users.

**Comment by verdie-g** [comment]
@yhmo note that we were considering waiting for .NET 11 for the built-in [BFloat16](https://learn.microsoft.com/en-us/dotnet/api/system.numerics.bfloat16)
